scripts/core/CreateConfigCode.py

```diff
@@ -233,7 +236,7 @@ def createHeader(c):
 #ifndef __CONFIG_OPTIONS_H__
 #define __CONFIG_OPTIONS_H__
 
-#include "Settings.h"
+#include <hoot/core/util/Settings.h>
```

Comment:
The `ConfigOptions.h` and `ConfigDefaults.h` files had a quoted `#include` and old copyright information so I updated them both.